smoke-test/src/test/java/org/opennms/smoketest/DaemonConfigPageIT.java

```diff
@@ -0,0 +1,258 @@
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+public class DaemonConfigPageIT extends OpenNMSSeleniumTestCase {
+    @Rule
+    public Timeout timeout = new Timeout(10, TimeUnit.MINUTES);
+
+    @Before
+    public void setUp() {
+        RestAssured.baseURI = getBaseUrl();
+        RestAssured.port = getServerHttpPort();
+        RestAssured.basePath = "/opennms/admin/daemons/index.jsp";
+        RestAssured.authentication = preemptive().basic(BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD);
+    }
+
+    @After
+    public void tearDown() {
+        RestAssured.reset();
+    }
+
+    @Test
+    public void verifyDaemonListIsShown() {
+        given().get().then()
+                .assertThat().statusCode(200)
+                .assertThat().contentType(ContentType.HTML);
+
+        DaemonReloadPage page = new DaemonReloadPage().open();
+        Assert.assertThat(page.getDaemonRows(), hasSize(30));
+
+        Daemon eventd = new Daemon(page, "eventd");
+        Daemon alarmd = new Daemon(page, "alarmd");
+        Daemon snmppoller = new Daemon(page, "snmppoller");
+        Daemon ticketer = new Daemon(page, "ticketer");
+
+        Assert.assertThat(eventd.getName(), is("eventd"));
+        Assert.assertThat(eventd.getStatus(), is("running"));
+        Assert.assertThat(eventd.isReloadable(), is(true));
+
+        Assert.assertThat(alarmd.getName(), is("alarmd"));
+        Assert.assertThat(alarmd.getStatus(), is("running"));
+        Assert.assertThat(alarmd.isReloadable(), is(true));
+
+        Assert.assertThat(snmppoller.getName(), is("snmppoller"));
+        Assert.assertThat(snmppoller.getStatus(), is("not running"));
+        Assert.assertThat(snmppoller.isReloadable(), is(false));
+
+        Assert.assertThat(ticketer.getName(), is("ticketer"));
+        Assert.assertThat(ticketer.getStatus(), is("running"));
+        Assert.assertThat(ticketer.isReloadable(), is(false));
+
+
+        Assert.assertThat(eventd.reload(5), is("Success"));
+        Assert.assertThat(eventd.getCurlString(), is(getExceptedCurlStringForDaemonName(eventd.getName())));
+
+        Assert.assertThat(alarmd.reload(5), is("Success"));
+        Assert.assertThat(alarmd.getCurlString(), is(getExceptedCurlStringForDaemonName(alarmd.getName())));
+    }
+
+    private String getExceptedCurlStringForDaemonName(String name){
+        return String.format("curl --request POST -u USERNAME:PASSWORD -url %sopennms/rest/daemons/reload/%s/", getBaseUrl(), name);
+    }
+
+    private class Daemon {
+        private final DaemonReloadPage page;
+        private final String daemonName;
+
+        public Daemon(DaemonReloadPage page, String daemonName) {
+            this.page = page;
+            this.daemonName = daemonName;
+        }
+
+        public String getName() {
+            // Assert That Right Name is shown
+            WebElement nameCell = getDaemonRowElement().findElements(By.xpath("./td[1]")).get(0);
```

Comment:
Any reason why `findElements` instead of `findElement` is used, if only one element is relevant anyways?
Also ensure that these calls are wrapped in a `setImplicitWait(5, SECONDS)`, otherwise execution takes very long if those elements are not visible due to changes or errors.